### PR TITLE
Add MKS Monster 8 generic config

### PR DESCRIPTION
**Pull request by Bomberski**
Adding MKS Monster 8 generic configuration

Signed-off-by: Allen Cox <Bomberski420@gmail.com>

### CODE REVIEW

File: config/generic-mks-monster8.cfg [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+#step_pin: PD13
+#dir_pin: !PD12
+#enable_pin: !PD14
+
+[heater_bed]
+heater_pin: PB10
+sensor_type: NTC 100K MGB18-104F39050L32
+sensor_pin: PC0
+max_power: 1.0
+control = pid
```
Comment (JamesH1978) [inline config/generic-mks-monster8.cfg:93]: Please change to : instead of =

Thanks
James

File: config/generic-mks-monster8.cfg [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+[stepper_x]
+step_pin: PC14
+dir_pin: PC13
+enable_pin: !PC15
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA14  # PA13 for X-max; endstop have'!' is NO
+position_endstop: 0
+position_max: 200
+homing_speed: 100
```
Comment (JamesH1978) [inline config/generic-mks-monster8.cfg:22]: Please leave homing_speed at default, some printers may not take kindly to being asked to home at twice default.

Thanks
James

File: config/generic-mks-monster8.cfg [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+[stepper_y]
+step_pin: PE5
+dir_pin: !PE4
+enable_pin: !PC15
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA15  # PC5 for Y-max; endstop have'!' is NO
+position_endstop: 0
+position_max: 200
+homing_speed: 100
```
Comment (JamesH1978) [inline config/generic-mks-monster8.cfg:33]: Same as above

File: config/generic-mks-monster8.cfg [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+[stepper_z]
+step_pin: PE1
+dir_pin: PE0
+enable_pin: !PE2
+microsteps: 16
+rotation_distance: 8
+endstop_pin: !PB13  # PB12 for Z-max; endstop have'!' is NO
+position_endstop: 0
+position_max: 220
+homing_speed: 25
```
Comment (JamesH1978) [inline config/generic-mks-monster8.cfg:44]: Same as above

**Review by JamesH1978 — CHANGES_REQUESTED** [review]
Looks fine to me, just the = to change, as remarked, plus the homing_speeds.

Thanks
James

**Comment by Bomberski** [comment]
I believe that's default sppeds

**Comment by JamesH1978** [comment]
`#homing_speed: 5.0
#   Maximum velocity (in mm/s) of the stepper when homing. The default
#   is 5mm/s.` :)

**Comment by Bomberski** [comment]
> ```
> #   Maximum velocity (in mm/s) of the stepper when homing. The default
> #   is 5mm/s.``` :)
> ```

I will swap from 50 to 5, thank you

**Comment by JamesH1978** [comment]
and i meant change = to : on all those option in heater_bed, and i edited my reply to say 50 is fine, it used in most example.

Thanks
James

**Review by JamesH1978 — APPROVED** [review]
Looks good to me

Thanks
James

**Comment by KevinOConnor** [comment]
Thanks.

-Kevin

**Pull request merged**